csrc/device_lower/analysis/non_divisible_split.cpp

```diff
@@ -42,6 +43,17 @@ void NonDivisibleSplitInfo::handle(Split* split) {
     return;
   }
 
+  // For 1D TMA, a domain parallelized by Bulk must not result from a
```

Comment:
I'm actually planning to deprecate this class. I'm not planning to add an alternative to `splits_to_validate_` since it doesn't actually guarantee anything.

Can you move it somewhere else?